features/bevragen/persoon-beperkt/naam/dev/autorisatie-gba.feature

```diff
@@ -30,7 +30,7 @@ Functionaliteit: Autorisatie voor naam in PersoonBeperkt
       En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
       | naam         | waarde |
       | afnemerID    | 000008 |
-      | gemeenteCode | 0800   |
+      | gemeenteCode | 800   |
```

Comment:
even een check. Waarom is in dit feature file de voorloopnul weggehaald en in de andere niet?

Reply:
foutje